doc/conf.py

```diff
@@ -0,0 +1,71 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
Is the idea to have (optionally) a conf.py in any repository that will have it's documentation created by sphinx-github-pages-generator? If sol, maybe add a comment here.

Reply:
It is a requirement for sphinx. i mentioned it in the user guide. this conf.py is just here because we also use sphinx to generate this documentation.

Reply:
Ok, then it's fine.